src/helpers/loadContract.ts

```diff
@@ -1,36 +1,43 @@
-const path = require('path')
-const replStarter = require('./replStarter')
-const { loadABI } = require('./utils')
-const Web3 = require('web3')
+import * as path from 'path'
+import Web3 from 'web3'
+import { Eth } from 'web3-eth';
+import { Contract } from 'web3-eth-contract';
```

Comment:
Req: There is some libraries not listed as dependency in package.json.
I think we can add a line in the `tslint` file with the rule `"no-unused-variable": true` and check for more issues like this.

Reply:
I'm going to disable the rule that forces you to list all dependencies in `package.json`, because we rely a lot on using `web3` sub-packages directly; it's even necessary to obtain some types.